src/app/operators/takeUntilDestroy.operator.ts

```diff
@@ -0,0 +1,11 @@
+import { ChangeDetectorRef, inject, ViewRef } from '@angular/core';
+import { Observable, ReplaySubject, takeUntil, UnaryFunction } from 'rxjs';
+
+export const takeUntilDestroy$ = <T>(): UnaryFunction<Observable<T>, Observable<T>> => {
```

Comment:
I know that variables that are Observables end with '$' and this is the finnish notation, but I'm not sure if we want to do this for operators and functions

Reply:
bump